index.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
+
```

Comment:
Yes, it only affects this file -- it is common practice and the more technical explanation is that Node actually wraps each "module" (file) in an anonymous function before executing its code (this can be seen with V8 debugging tools such as [node-inspector](https://github.com/node-inspector/node-inspector).

The main purpose of Strict Mode is to [convert mistakes into errors](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Converting_mistakes_into_errors) (e.g. assigning to undeclared vars throw errors, as well as duplicated property names on an object definition), thus the developer can spot these errors more easily. For instance, you would have spotted [this missing variable declaration](https://github.com/pkrumins/node-tree-kill/pull/10/files#diff-168726dbe96b3ce427e7fedce31bb0bcL89), as the code would throw an error in strict mode.

There are few breaking edge cases -- namely, involving [`eval` and `arguments`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Making_eval_and_arguments_simpler), `this` no longer implicit references the global/module object for functions called via bare function calls (e.g. `fn()`). As far as I see, none of these affect the behavior of this module and the unit tests are still fine.

I can remove this if you feel uncomfortable, though.